libfwupd/fwupd-enums.h

```diff
@@ -1093,23 +1093,23 @@ typedef enum {
 	/**
 	 * FWUPD_INSTALL_FLAG_IGNORE_CHECKSUM:
 	 *
-	 * Ignore firmware CRCs and checksums.
+	 * This is now unused; see #FuFirmwareParseFlags.
```

Comment:
I think add this to a list of things to drop on the next ABI bump

Reply:
Good idea, added in a follow up commit. Thanks.